jupyter-on-gke/README.md

```diff
@@ -98,11 +105,18 @@ We allow user to set their own domains, in the `variables.tf` file. Since we are
 
 ### If Auth is disabled
 
+For demo purposes, this disabling auth creates a public IP for the Jupyter notebook with basic dummy authentication. To secure your cluster, it is _strongly recommended_ to enable authenticator and use IAP.
+
+For more information, please take a look at the following links:
+* https://cloud.google.com/iap/docs/enabling-kubernetes-howto
+* https://cloud.google.com/endpoints/docs/openapi/get-started-kubernetes-engine
+* https://jupyterhub.readthedocs.io/en/stable/tutorial/getting-started/authenticators-users-basics.html
+
 1. Run `kubectl get services -n <namespace>`. The namespace is the user name that you specified above.
 
-2. Copy the external IP for the notebook.
+2. Copy the external IP for the notebook. If there isn't an external IP for `proxy-public`, is it most likely due to authencation being enabled.
 
-Continue to Step 3.
+Continue to Step 3 of [below](#if-auth-is-enabled).
 
 ### If Auth is enabled
```

Comment:
Need to delete the "Securing endpoints" section below

Reply:
Done!